recipes/python-irodsclient/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "python-irodsclient" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "ca4fb5c390744420d2265043796a8fbb5f9ca1e3ac2c19e13e3072fc52161daf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```

Comment:
It looks like this package requires `PrettyTable`:

https://github.com/irods/python-irodsclient/blob/5f23b41cb0a619739d4b876071425614588f37e8/setup.py#L30